static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -20,252 +20,254 @@
+    **/
+    background-position: 49% 49%;
+    background-repeat: no-repeat;
+    z-index: 1;
+    text-align: center;
+}
+
+.youtube-media-atom__overlay .video-overlay {
+    text-align: initial;
+}
+
+.youtube-media-atom__play-button.vjs-control-text {
+    overflow: hidden !important;
+    display: inline-block;
+    z-index: 2;
+}
+
+.youtube-media-atom__bottom-bar {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 34px;
+    z-index: 2;
+}
+
+.youtube-media-atom__bottom-bar::before,
+.youtube-media-atom__bottom-bar::after {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    left: 0;
+    z-index: -1;
+}
+
+.youtube-media-atom__bottom-bar::before {
+    content: '';
+    opacity: .1;
+    background: $neutral-1;
+}
+
+.youtube-media-atom__bottom-bar::after {
+    @include simple-gradient(rgba(51, 51, 51, 0), rgba(51, 51, 51, .5));
+    content: '';
+    opacity: .7;
+}
+
+.youtube-media-atom__bottom-bar__icon {
+    float: left;
+    height: 100%;
+    width: 36px;
+}
+
+.youtube-media-atom__bottom-bar__icon svg {
+    fill: $media-default;
+    width: 19px;
+    height: 12px;
+    top: auto;
+    bottom: 10px;
+    left: 10px;
+}
+
+.youtube-media-atom__bottom-bar__duration {
+    @include font($f-sans-serif-text, bold, 16, 30);
+    color: $media-default;
+    float: left;
+    -webkit-font-smoothing: subpixel-antialiased;
+    padding-top: 4px;
+    padding-left: 8px;
+    text-shadow: 0 0 16px rgba(51, 51, 51, .25);
+}
+
+.youtube-media-atom__bottom-bar__duration::before {
+    content: '';
+    width: 1px;
+    height: 16px;
+    opacity: .5;
+    position: absolute;
+    left: 36px;
+    background-color: $multimedia-main-2;
+    top: auto;
+    bottom: 8px;
+}
+
+.immersive-main-media__media .youtube-media-atom {
+    padding-bottom: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    height: 100%;
+    z-index: 0;
+}
+
+.immersive-main-media__media  .youtube-media-atom__immersive-interface {
+    position: absolute;
+    top: 50vh;
+    transform: translateY(-100%);
+    padding-bottom: $gs-baseline * 2;
+}
+
+.immersive-main-media__media  .youtube-media-atom__play-button,
+.immersive-main-media__media  .youtube-media-atom__bottom-bar__duration {
+    display: inline-block;
+    vertical-align: middle;
+    float: none;
+    top: auto;
+    left: auto;
+    right: auto;
+    bottom: auto;
+}
+
+.immersive-main-media__media .youtube-media-atom__play-button {
+    position: relative;
+    width: 92px;
+    height: 92px;
+    transform: none;
+}
+
+.immersive-main-media__media .youtube-media-atom__play-button:hover {
+    transform: none;
+}
+
+.immersive-main-media__media .youtube-media-atom__play-button:before {
+    width: 100%;
+    height: 100%;
+    border-radius: 100%;
+}
+.immersive-main-media__media .youtube-media-atom__play-button:after {
+    display: none;
+}
+
+.immersive-main-media__media .youtube-media-atom__play-button svg {
```

Comment:
InlineSvg takes a third optional param so you can do something like this: `@fragments.inlineSvg("play", "icon", List("my-class-name"))`

Then the svg will have `my-class-name__svg` as a class name